src/server/lock_manager/waiter_manager.rs

```diff
@@ -117,11 +118,11 @@ pub enum Task {
         deadlock_key_hash: u64,
     },
     ChangeConfig {
-        timeout: Option<u64>,
-        delay: Option<u64>,
+        timeout: Option<ReadableDuration>,
+        delay: Option<ReadableDuration>,
     },
     #[cfg(any(test, feature = "testexport"))]
-    Validate(Box<dyn FnOnce(u64, u64) + Send>),
+    Validate(Box<dyn FnOnce(ReadableDuration, ReadableDuration) + Send>),
```

Comment:
Please also change `Task::Validate` in `deadlock.rs` take `Validate(Box<dyn FnOnce(ReadableDuration) + Send>)` like this one.